### PR TITLE
Feat/improve prompt and schema reading for bull flow

### PR DESCRIPTION
**Pull request by hugo-nl**
🎸 Dynamic reading of prompt and schema --> We can know make minor changes to them without having to rerun the entire flow for it to take effect

### CODE REVIEW

**Review by Greenheart — CHANGES_REQUESTED** [review]
This will be a very nice improvement! Good find and fix :)

Some minor comments, and a merge conflict to resolve before we merge.

File: src/workers/extractEmissions.ts [hunk 1]
```diff
@@ -21,11 +16,13 @@ const flow = new FlowProducer({ connection: redis })
 const extractEmissions = new DiscordWorker<JobData>(
   'extractEmissions',
   async (job) => {
-    const { companyName } = job.data
+    const { companyName, type } = job.data
     job.sendMessage(`🤖 Hämtar utsläppsdata...`)
 
     const childrenValues = await job.getChildrenEntries()
 
+    console.log('type1111', type)
```
[the remark this answers is not in the document]
Comment (hugo-nl) [reply src/workers/extractEmissions.ts]: Yeah sorry I wasnt ready for review quite yet 😄

File: src/types/Company.ts [hunk 2]
```diff
@@ -23,3 +23,18 @@ export type Scope2 = {
...
+export enum JobType {
+  IndustryGics = 'followUp/industry_gics',
+  Scope12 = 'followUp/scope12',
+  Scope3 = 'followUp/scope3',
+  Biogenic = 'followUp/biogenic',
+  Economy = 'followUp/economy',
+  Goals = 'followUp/goals',
+  Initiatives = 'followUp/initiatives',
+
+  FiscalYear = 'fiscalYear',
```
[the remark this answers is not in the document]
Comment (hugo-nl) [reply src/types/Company.ts:36]: Great!

**Review by Greenheart — APPROVED** [review]
Well done! :tada:

**Pull request merged**